lib/backburner.js

```diff
@@ -8,15 +8,14 @@ import {
   now
 } from './backburner/utils';
 
-import platform, {
```

Comment:
If we are updating Ember to only use Esperanto, I'd prefer to keep this syntax (instead of two imports).